src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -350,7 +350,7 @@ object TezosDatabaseOperations extends LazyLogging {
         ids =>
           writeAccountsCheckpoint(
             List(
-              (hash, level, Some(timestamp), cycle, ids.map(AccountId(_)).toList)
+              (hash, level, Some(timestamp), cycle, None, ids.map(AccountId(_)).toList)
```

Comment:
aren't we losing info here?

Reply:
ok,but I can't clearly make out, why we need to pass None if that parameter is not needed anyway? Can't it be removed completely from the expected values at this point in the code?

Reply:
but in one place we use `.asTuple` to extract this data for passing to the `writeAccountsCheckpoint`, and here we're not.

Reply:
ok, now my eyes are opened... I can see clearly :eye: